src/jsep.js

```diff
@@ -515,7 +501,9 @@ export class Jsep {
 		}
 
 		if (!node) {
```

Comment:
Was that supposed to be `this.node`?

Reply:
I think it would have worked -- but I'll push a change to use env.node in all cases here